Multiset/Multiset.playground/Sources/Multiset.swift

```diff
@@ -7,18 +7,24 @@
 
 import Foundation
 
-public struct Multiset<Element: Hashable> {
-  private var storage: [Element: UInt] = [:]
+public struct Multiset<T: Hashable> {
+  private var storage: [T: UInt] = [:]
   public private(set) var count: UInt = 0
 
   public init() {}
 
-  public mutating func add (_ elem: Element) {
+  public init<C: Collection>(_ collection: C) where C.Element == T {
```

Comment:
Great addition. I propose we also add a little Swift syntactical sugar by adopting the `ExpressibleByArrayLiteral` protocol:

```
extension Multiset: ExpressibleByArrayLiteral {
  public init(arrayLiteral elements: T...) {
    self.init(elements) // using the `Collection` initializer you created
  }
}
```

This allows us to write code like this:

```
let multiset: Multiset<String> = ["Cacti", "Tactical"]
```

Reply:
😲 😲 😍